pyproject.toml

```diff
@@ -8,6 +8,33 @@ version = "1.1.0"
 description = "The awesome Fortinet Toolbox"
 authors = ["Patrik Spiess <patrik.spiess@mgb.ch>", "Lukas Murer-Jäckle <lukas.murer@mgb.ch>"]
 readme = "README.md"
+license = "LGPL-3.0-only"
+repository = "https://github.com/migros/fotoobo"
+classifiers = [
+    "Framework :: Pytest",
+    "Framework :: Sphinx",
+    "Framework :: Sphinx :: Extension",
+    "Framework :: Sphinx :: Theme",
+    "Framework :: tox",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: System Administrators",
+    "Intended Audience :: Telecommunications Industry",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "Natural Language :: English",
+    "Operating System :: Unix",
```

Comment:
Is there no explicit "Linux" here? Perhaps we will be found better with adding this too

Reply:
:+1: